Description: Correct pod NAME issue in LedgerSMB/IR.pm
 Is missing a pod NAME entry altogether.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472604&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/IR.pm
+++ b/LedgerSMB/IR.pm
@@ -1,3 +1,11 @@
+=pod
+
+=head1 NAME
+
+LedgerSMB::IR - Inventory received module
+
+=cut
+
 #=====================================================================
 # LedgerSMB
 # Small Medium Business Accounting software
